src/data/faq_de.json

```diff
@@ -102,6 +102,159 @@
+                    },
+                    {
+                        "title": "Kann ich den QR-Code auch ohne Internetverbindung vorzeigen?",
+                        "anchor": "vaccination_certificate_internet",
+                        "active": true,
+                        "textblock": [
+                            "Ja. Sobald der QR-Code in der Corona-Warn-App angezeigt wird, können Sie ihn jederzeit ohne eine bestehende Internetverbindung vorzeigen."
+                        ]
+                    },
+                    {
+                        "title": "Bin ich verpflichtet, den digitale Impfnachweis in der Corona-Warn-App zu nutzen? Wird es trotzdem noch den gelben analogen Impfausweis geben? Kann ich meinen gelben Impfpass auch weiterhin benutzen? Was ist mit Personen, die kein Smartphone besitzen?",
+                        "anchor": "vaccination_certificate_voluntary",
+                        "active": true,
+                        "textblock": [
+                            "Der digitale Impfnachweis per Corona-Warn-App ist lediglich ein vollkommen freiwilliges und ergänzendes Angebot. Wenn Sie keinen digitalen Impfnachweis besitzen oder diesen verloren haben, ist der Impfnachweis über das bekannte „gelbe Heft“ (gelber Impfpass) weiterhin möglich und gültig. Ausserdem können Sie Corona-Impfungen auch mit dem ausgedruckten Corona-Impfzertifikat für die vollständige Impfung nachweisen."
+                        ]
+                    },
+                    {
+                        "title": "Wie soll der Nachweis einer Impfung geprüft werden können?",
+                        "anchor": "vaccination_certificate_verification",
+                        "active": true,
+                        "textblock": [
+                            "Das Impfzertifikat (QR-Code) wird z.B. über die CovPass App oder die Corona-Warn-App (CWA) digital oder alternativ als maschinenlesbarer Ausdruck genutzt. Das Impfzertifikat enthält nur Informationen zum Impfstatus, den Namen des Geimpften und das Geburtsdatum. Für Dienstleister, die den Impfstatus überprüfen möchten, wird es eine Prüf-App zur Prüfung des Impfzertifikats geben. Damit kann der Impfstatus ähnlich wie ein Barcode eines Flug- oder Bahntickets gescannt werden. Alternativ ist ein Nachweis mit dem analogen Impfpass möglich."
+                        ]
+                    },
+                    {
+                        "title": "Kann man auch im digitalen Impfnachweis speichern, dass man bereits infiziert war oder negativ getestet wurde?",
+                        "anchor": "vaccination_certificate_recovered_negative_test",
+                        "active": true,
+                        "textblock": [
+                            "Ja, auch negative Tests oder eine durchgemachte Infektion lassen sich in der CovPass- App und Corona-Warn-App als Testzertifikat bzw. Genesenenzertifikat hinterlegen. Eine durchgemachte Infektion wird mittels eines PCR-Test-Ergebnisses dokumentiert."

```

Comment:
Same question as in English section: Is this already possible in CWA 2.3?

Reply:
vac_cert_recovered_negative_test

Reply:
removed